### ISSUE TITLE
Wrong Headers scrollview size

### ISSUE THREAD

**Issue by vg-identance** [kean/Pulse#27]
Found issue with scrolling in `Headers` tab. Seems that content size of scrollview is much bigger than the real content size (about 20-30%). 

iPhone X, iOS 14.6, presented `MainViewController`. 100% reproduce on all requests.

<img src="https://user-images.githubusercontent.com/49786280/126645035-98c68e72-fa1e-40af-9f2b-5b74710f31a7.jpeg" alt="IMG_70CFE23B1589" width="400">

**Comment by kean** [comment]
Thanks for the report. Yeah, I saw a couple of UI issues with `MainViewController`. I would suggest to use `MainView` (SwiftUi component directly if possible). I would also appreciate PRs.

**Comment by vg-identance** [comment]
So I debugged it a bit. Reproduced in your Mac Pulse app. I could say that if remove next line https://github.com/kean/Pulse/blob/26ecf9936b851b061dde433060b86ecb10d2c8d3/Pulse/Sources/PulseUI/Views/KeyValueSectionView.swift#L54 issue is gone. Probably could be replaced with some static vertical `padding` But I'm not a guru of SwiftUI (yet) so not sure that this is the best solution so decided not to make PR.
